### PR TITLE
feat(SelectDropdown): Add ability to add label to select dropdown

### PR DESCRIPTION
**Pull request by ayeshakmaz**
Needed for [Frontend PR #11841](https://github.com/optimizely/optimizely/pull/11841)

JIRA:
[UXENG-56](https://optimizely.atlassian.net/browse/UXENG-56)

### CODE REVIEW

File: src/components/Input/index.scss [hunk 1]
```diff
@@ -23,14 +23,17 @@
...
-  height: spacer(3.5);
 }
 
 .#{$namespace}text-input[type="number"],
 .oui-text-input[type="number"] {
   text-align: right;
 }
 
+.oui-text-input[type="time"] {
+  max-height: map-fetch($button, size, base, height);
```
Comment (jessicalc) [inline src/components/Input/index.scss]: How come the time input has a max-height explicitly set, but the other inputs don't?

File: src/components/SelectDropdown/SelectDropdown.story.js [hunk 2]
```diff
@@ -146,7 +146,7 @@ stories.add('Default', (() => {
       />
     </Container>
   );
-})).add('ActivatorLabel', (() => {
+})).add('Activator Label differs from Value', (() => {
```
Comment (jessicalc) [inline src/components/SelectDropdown/SelectDropdown.story.js]: Thanks for making this clearer!

File: src/components/SelectDropdown/index.js [hunk 3]
```diff
@@ -8,6 +8,12 @@ import Link from '../Link';
 
 class SelectDropdown extends React.Component {
   static propTypes = {
+    /**
+     * Button text object with label and current value
```
Comment (jessicalc) [inline src/components/SelectDropdown/index.js]: I think you mean "content" value but maybe not? 
```suggestion
     * Button text object with label and content value
```

File: src/components/Input/index.scss (hunk 1 above)
Comment (ayeshakmaz) [reply src/components/Input/index.scss]: The native, out-of-the-box time input is slightly larger due to the arrow buttons, so this forces it to be equal to our other inputs (it's a very annoying HTML thing :P )

**Pull request merged**